src/components/templates/PartyMap/PartyMapRouter.tsx

```diff
@@ -13,7 +11,6 @@ export const PartyMapRouter: FC = () => {
 
   return (
     <Switch>
-      <Route exact path={`${match.path}/admin`} component={BannerAdmin} />
       <Route path={`${match.path}/:roomTitle`} component={PartyMap} />
       <Route path={`${match.path}/`} component={PartyMap} />
     </Switch>
```

Comment:
To be honest, I can't see this `/:roomTitle` anywhere in partymap. Clicking on a room show a modal and not a route. I think I can safely remove this one.

Reply:
I don't think it can be removed. I remember that we had a bug a while back and the fix was adding this route.

I believe this is used when it's a sub-venue like a jazz bar.

Clicking a room will open the modal, but entering a room (that isn't a link off-platform like zoom) will use this route.

Reply:
I think we can probably treat this question as OOS for this PR.

If we moved it to `AppRouter` as-is, then it will enable that route even for templates that don't support subvenues, which feels wrong.

It's not a huge issue still having these 'child routers' here for now.